gym/utils/play.py

```diff
@@ -161,7 +164,19 @@ def callback(obs_t, obs_tp1, action, rew, done, info):
         Random seed used when resetting the environment. If None, no seed is used.
     """
     env.reset(seed=seed)
-    game = PlayableGame(env, keys_to_action, zoom)
+
+    key_code_to_action = {}
+
+    for key_combination, action in keys_to_action.items():
+        key_code = tuple(
+            sorted(ord(key) if isinstance(key, str) else key for key in key_combination)
+        )
+        key_code_to_action[key_code] = action
+
+    game = PlayableGame(env, key_code_to_action, zoom)
+
+    if fps is None:
+        fps = env.metadata.get("render_fps", 30)
```

Comment:
I think some environments don't have "render_fps", wrapper `monitoring.video_recorder` provides some code on this

Reply:
True, but I think it's fine as long as I use `get`, right? I think that's also how `monitoring.video_recorder` does it :)

Reply:
That is true, plus we say in the documentation that the default is 30.